tests/testthat/test-nonmem.R

```diff
@@ -29,14 +29,12 @@ test_that("NONMEM dsl", {
     rxToNonmem(x, ui)
   }
 
+  # Function Definitions ####
   expect_equal(.rxToN("sqrt(a)"), "DSQRT(RXDZ001)")
   expect_equal(.rxToN("max(a,b)"), "MAX(RXR1,RXR2)")
   expect_equal(.rxToN("max(c,b,a)"), "MAX(RXR3,RXR2,RXR1)")
   expect_equal(.rxToN("sum(a,b,c,d)"), "((RXR1)+(RXR2)+(RXR3)+(RXR4))")
   expect_equal(.rxToN("prod(a,b,c,d)"), "((RXR1)*(RXR2)*(RXR3)*(RXR4))")
```

Comment:
Spacing does matter for NONMEM, I can't recall, but I think it might.  I was thinking about removing whitespace for these tests.

Reply:
Mostly, spacing doesn't matter in NONMEM.  There are some issues with indentation tracking that I need to track down.  I'd prefer to make it consistent and fix the spacing issues.  (I prefer to have somewhat readable code generated.)